### DISCUSSION TITLE
Support for Twitter/X advertising/sponsored Tweets/X

### DISCUSSION THREAD

**Discussion by theorzr** [uBlockOrigin/uBlock-issues#3087] [Ideas]
I'm wondering if this was ever discussed to block twitter/x sponsored posts?

**Comment by ghost** [comment]
https://www.reddit.com/r/uBlockOrigin/wiki/solutions/twitter

**Comment by ghost** [comment]
Or do you mean these ad posts not marked as ads? https://adguard.com/en/blog/twitter-x-ads-unlabelled-musk.html

**Reply by theorzr** [reply]
Ok, maybe I'm wrong on this because it seems to already be implemented. I posted this after getting half of my timeline filled with advertised tweets, but this was from the mobile app and I was sure that it was the case on the website as well, but I was wrong.

**Reply by gorhill** [reply]
> was from the mobile app and I was sure that it was the case on the website as well

Twitter ads are blocked by default in uBO.
